libs/langchain-weaviate/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@langchain/weaviate",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
In the future please don't change version numbers in PRs. We communicate the type of change via [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) messages, and then use these to decide how to modify version numbers when we release.